templates/tutorialv2/index.html

```diff
@@ -34,6 +43,9 @@
     {% elif articles != None %}
         <li><a href="{% url "article:list" %}">{% trans "Articles" %}</a></li>
         <li><a href="{% url "content:find-article" usr.pk %}">{% trans "Articles de" %} {{ usr.username }}</a></li>
+    {% elif opinions != None %}
+        <li><a href="{% url "opinion:list" %}">{% trans "Tribunes" %}</a></li>
+        <li><a href="{% url "content:find-opinion" usr.pk %}">{% trans "Tribune de" %} {{ usr.username }}</a></li>
```

Comment:
On n'a pas besoin de vérifier si la tribune est validé ou non ? ([Ligne 64](https://github.com/zestedesavoir/zds-site/pull/3606/files#diff-6210defe7c031d7578cfb9f6b5a81bc5R64) même question)

    {% elif opinions != None %}
        {% if current_filter.key != 'validation' and current_filter.key != 'beta' %}

Comme pour la [ligne 158](https://github.com/zestedesavoir/zds-site/pull/3606/files#diff-6210defe7c031d7578cfb9f6b5a81bc5R158) ?